_episodes/01-quickstart.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Quickstart guide"
+teaching: 2
+exercises: 8
+questions:
+
+- "What is the purpose of the quickstart guide?"
+- "How do I load and check the ESMValTool environment?"
+- "How do I configure ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Understand the purpose of the quickstart guide"
+- "Load and check the ESMValTool environment"
+- "Configure ESMValTool"
+- "Run a recipe"
+
+keypoints:
+
+- "The purpose of the quickstart guide is to enable a user of ESMValTool
+  to run ESMValTool as quickly as possible"
+- "Use the `module load` command to load the ESMValTool environment 
+  (see the installation guide for more details) and use `esmvaltool --help` to 
```

Comment:
Does the "installation guide" here refer to the installation episode or the Read the Docs page on ESMValTool installation? Perhaps you could add a hyperlink to the guide.

Reply:
Changed it to installation episode and added link 1c3cfb5.